src/xx-apt

```diff
@@ -74,14 +74,14 @@ case "${TARGETARCH}" in
   riscv*)
     case "${XX_VENDOR}" in
       debian)
-        if ! grep "sid main" "$(aptsourcesfile)" >/dev/null; then
-          echo >&2 "skipping packages installation on ${XX_ARCH}. riscv64 currently only supported on sid"
+        if [ "$(cut -d. -f 1 /etc/debian_version)" -lt 13 ]; then
```

Comment:
Let's add force override as well in case the packages are different from distro. Can be one override for all distros..